kie-internal/src/main/java/org/kie/internal/runtime/manager/context/CaseContext.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2013 JBoss Inc
```

Comment:
updated header

P.S.
yeah, I have plenty of these changes to "release to public" that I worked on them for years... ;)